lib/Config.js

```diff
@@ -14,10 +14,12 @@ class Config {
      * This is the default constructor of the Config Object, and the only way
      * to instanciate it (with default parameters).
      *
+     * @param {object} config     - configuration settings to create/update with
      * @returns {undefined}
      */
-    constructor() {
+    constructor(config) {
```

Comment:
indeed, this is a good way to allow defining one config when instanciating the `API`, rather than force the use of `API.reconfigure(config)`